packages/components/src/components/chat-search/Context.vue

```diff
@@ -1,9 +1,8 @@
 <template>
   <div class="context" data-cy="context">
-    <template v-if="!contextResponse && !hasPinnedItems">
+    <template v-if="!contextResponse">
       <div class="context__notice-container">
```

Comment:
The bottom area of the context notice is clipped
![image](https://github.com/user-attachments/assets/7f0d60f9-1097-4938-a203-ccb27ab42316)